Parsers/ASimAuthentication/Parsers/ASimAuthenticationSudo.yaml

```diff
@@ -0,0 +1,106 @@
+Parser:
+  Title: Authentication ASIM parser for Syslog sudo
+  Version: '0.1.1'
+  LastUpdated: November 21, 2022
+Product:
+  Name: sudo
+Normalization:
+  Schema: Authentication
+  Version: '0.1.1'
+References:
+- Title: ASIM Authentication Schema
+  Link: https://aka.ms/ASimAuthenticationDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
+Description: |
+  This ASIM parser supports normalizing Syslog sudo sign in logs to the ASIM Authentication schema.
+ParserName: ASimAuthenticationSudo
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let SudoSignInAuthorized=(disabled:bool=false){
+  Syslog | where not(disabled)
```

Comment:
This actually has to be added as a ProcessEvent parser to be effective in analytics. I am contemplating whether here the same event needs to be parser or this one:

`pam_unix(sudo:session): session opened for user root by (uid=0)`

Reply:
I'm not sure that I understand.
You want to link the ProcessEvent with the AuthenticationEvent?